src/Data/Connection/CommentConnectionResolver.php

```diff
@@ -245,4 +245,21 @@ public function sanitize_input_fields( array $args ) {
 
 	}
 
+	/**
+	 * Determine whether or not the the offset is valid, i.e the comment corresponding to the offset exists.
+	 * Offset is equivalent to comment_id. So this function is equivalent
+	 * to checking if the comment with the given ID exists.
+	 *
+	 * @return bool
+	 */
+	public function is_valid_offset( $offset ) {
+		global $wpdb;
+
+		if ( ! empty( wp_cache_get( $offset, 'comment' ) ) ) {
+			return true;
+		}
+
+		return $wpdb->get_var( $wpdb->prepare( "SELECT EXISTS (SELECT 1 FROM $wpdb->comments WHERE comment_ID = %d)", $offset ) );
```

Comment:
Is this multisite supported? Besides, where are you caching the offset?

Reply:
@renatonascalves ah! I actually reviewed the PR this was part of before this commit came in, and I guess I overlooked this commit's changes. (https://github.com/wp-graphql/wp-graphql/pull/1111/commits/25e5812876359b428882f10af2161eb5867db3a3#r366142346)

Thanks for pointing this out. I've reverted these changes as they actually won't be more performant at this point because the cursor objects are cached when the query first starts, so just asking for the objects again will be faster than another raw sql query. And, as you point out, the cache wasn't being set anyway.